### ISSUE TITLE
Different Project Names for branches

### ISSUE THREAD

**Issue by DoCode** [erikzielke/youtrack-jenkins#2]
I have for each git branch a separate Jenkins job (eg Foo.master, Foo.develop, ...)
Where can a Jenkins job assigned to a YouTrack project?

**Comment by erikzielke** [comment]
I am not sure I understand. Commands in the SCM message are just applied to the issue, e.g. yp-123. 

Do you want to limit which projects commands can be applied to per job, and should this limit also apply for changelog annotations? If that is want you want, this is not currently possible.

There is another issue with multiple branches, which is that commit messages will be seen several times, whenever branches are merged.

**Comment by DoCode** [comment]
Ok, I understand...
I must enable
**`Execute commands`**
and write in commit message
**`#Prefix-1234 fixed`**
then the Issue Prefix-1234 marked as fixed.
Is this possible?

**Comment by erikzielke** [comment]
Yes, this is possible. But as I mentioned above the command will be applied multiple times, if you e.g. on dev branch has the message #prefix-1234 Fixed, this will be applied in you dev Jenkins job, then when those changes are merged to prod branch, then the prod Jenkins job will apply the command again.
